improving.html

```diff
@@ -154,14 +154,14 @@ <h2 id="scope">What is your Scope?</h2>
-	<p>You may need to define a phased approach with different dates for different levels. For example, meet particular WCAG 2.0 success criteria in the next release, and meet all Level A and Level AA success criteria in the following release.</p>
-	<p>Note that in some cases, some Level AAA success criteria may be fairly easy to meet. For example, refining appropriate link text (2.4.9, Level AA) and heading structure (2.4.10, Level AAA) may be easy to address together when revising content.</p>
+	<p>You may need to define a phased approach with different dates for different levels. For example, meet particular WCAG 2.0 success criteria in the next release, and meet all <a href="https://www.w3.org/WAI/WCAG20/quickref/?currentsidebar=%23col_overview&levels=aaa">Level A and Level AA</a> success criteria in the following release.</p>
+	<p>Note that in some cases, some Level AAA success criteria may be fairly easy to meet. For example, refining appropriate link text (2.4.4, Level A) and heading structure (2.4.10, Level AAA) may be easy to address together when revising content.</p>
 
 	<h2 id="repair">What is your Strategy?</h2>
 	<p>Consider the following tips to help you implement accessibility repairs:</p>
 	<ul>
 		<li><strong>Leverage the different skills in your team</strong> &mdash; While many tasks will be for developers, other roles have plenty to contribute. For example, designers can select better colors and content authors can improve the wording of links, headings, and text alternatives. The resource <a href="https://www.w3.org/WAI/eval/reviewteams.html">Using Combined Expertise to Evaluate Web Accessibility</a> may be helpful.</li>
-		<li><strong>Communicate requirements across your team</strong> &mdash; Ensure that everyone involved in repairs understands the basics of web accessibility and the specific requirements they need to address. Distribute the <a href="resources">Important Resources</a> to the relevant members of your team.</li>
+		<li><strong>Communicate requirements across your team</strong> &mdash; Ensure that everyone involved in repairs understands the basics of web accessibility and the specific requirements they need to address. Distribute the <a href="#resources">Important Resources</a> to the relevant members of your team.</li>
 		<li><strong>Validate solutions as early as possible</strong> &mdash; Ensure that any solutions adequately address the issues raised to avoid implementing changes that do not work in practice. If at all possible, it is important to involve people with disabilities in such validation. The resources <a href="https://www.w3.org/WAI/users/involving">Involving Users in Web Projects for Better, Easier Accessibility</a> and <a href="https://www.w3.org/WAI/eval/users.html">Involving Users in Evaluating Web Accessibility</a> provide more background.</li>
```

Comment:
Changed to "The resources about <a href="https://www.w3.org/WAI/users/involving">involving users in web projects</a> and <a href="https://www.w3.org/WAI/eval/users.html">evaluating with users</a> provide more background."